pages/business/setup/keys.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from 'react'
 import Head from 'next/head'
-
 import { isFormal } from '~lib/config'
 import { WithOwnerProps, withValidPrivateKey } from '~lib/pageWrappers'
-import { Text, Box, ButtonLink } from '~ui/core'
+
+import { Text, Box, ButtonLink, Button } from '~ui/core'
 import { ArrowsRight } from '~ui/anicons'
 import { MobileApp } from '~ui/layouts/MobileApp'
 import { KeyViewer } from '~ui/blocks/KeyViewer'
 
 const SetupKeysPage: React.FC<WithOwnerProps> = ({ owner }) => {
   const { privateKey } = owner
+  const css_inl = '@media print { .no-print{ display: none !important;} }'
```

Comment:
Although I'm personally not a fan of it, but we're using [Styled Components](https://emotion.sh/docs/styled)
So CSS is not done like this inline, but using the `styled(...)` function. Example here: https://github.com/railslove/rcvr-app/blob/master/pages/business/setup/verify-key.tsx#L144-L149